common/src/main/java/org/apache/falcon/persistence/PersistenceConstants.java

```diff
@@ -80,5 +80,6 @@ private PersistenceConstants(){
     public static final String GET_ALL_EXTENSION_JOBS = "GET_ALL_EXTENSION_JOBS";
     public static final String DELETE_EXTENSION_JOB = "DELETE_EXTENSION_JOB";
     public static final String GET_EXTENSION_JOB = "GET_EXTENSION_JOB";
+    public static final String GET_JOBS_FOR_AN_EXTENSION = "GET_JOBS_FOR_AN_EXTENSION";
```

Comment:
Was thinking the same initially, but then to get all details I can always create a new query. I have made the changes accordingly.